chromium_src/chrome/browser/ui/views/incognito_clear_browsing_data_dialog.cc

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
```

Comment:
Good job! You found it BubbleDialogDelegateView also has ShowCloseButton declaration 👍

Reply:
Thank you, could not have figured it out without your help 🙏 
I've also fixed up the new line stuff 💪